src/com/phronemophobic/membrane/term.clj

```diff
@@ -340,14 +340,34 @@
    :background      [1     1     1]
    :foreground      [0     0     0]})
 
-(def default-common-opts {:width 90
-                          :height 30
-                          :font-family :monospace
-                          :font-size 12
-                          :toolkit nil
-                          :color-scheme default-color-scheme})
+(def ^:private default-common-opts {:width 90
+                                    :height 30
+                                    :font-family :monospace
+                                    :font-size 12
+                                    :toolkit nil
+                                    :color-scheme default-color-scheme})
 
 (defn run-term
+  "Launch an interactive membrane.term terminal. Terminal exits when explicitly closed by user.
+
+  Accepts optional `opts` map:
+  - `:width`         Window width in characters (default: `90`)
+  - `:height`        Window height in characters (default: `30`)
+  - `:font-family`   OS installed font family name. Example: `\"Courier New\"`.
+     Use `:monospace` for default monospace (default: `:monospace`)
+  - `:font-size`     Font point size (default: `12`)
+  - `:toolkit`       Graphics toolkit, specify `\"java2d\"` or `\"skia\"` (default: `\"java2d\"`)
```

Comment:
toolkit is not a string. `toolkit` should be an object that satisfies the following interfaces from `membrane.toolkit`:

IToolkit
IToolkitLogicalFontFontFamily
IToolkitFontExists
IToolkitFontMetrics
IToolkitFontAdvanceX
IToolkitFontLineHeight
IToolkitRunSync

And for screenshot:
IToolkitSaveImage

Reply:
sacre bleu! I shall update.